Vungle/VungleAdapterConfiguration.m

```diff
@@ -45,7 +47,11 @@ - (NSString *)adapterVersion {
 }
 
 - (NSString *)biddingToken {
-    NSString *bidToken = [[VungleRouter sharedRouter] currentSuperToken];
+    NSString *token = [[VungleRouter sharedRouter] currentSuperToken];
+    if (token.length) {
+        bidToken = token;
+    }
+    MPLogInfo(@"Vungle: Get bid token: %@.", bidToken);
     return bidToken;
```

Comment:
can't u use this instead of the static string?? i do not see any other usage of this here.

return [[VungleRouter sharedRouter] currentSuperToken].length > 0 ? [[VungleRouter sharedRouter] currentSuperToken]:nil;

Reply:
I like the idea of calling [currentSuperToken] just once and saving it in the local variable since it is accessed twice, especially since that method is not a simple getter.

Reply:
👍